### DISCUSSION TITLE
Getting freezed after turning on any vpn app(m1 mac)

### DISCUSSION THREAD

**Discussion by Esiou** [Genymobile/feedback#57] [Genymotion Desktop] [answered]
I've used several vpns but the problem still exists. Right after hitting the ok button for permission, it freezes. 
version 3.4 - macbook air m1

**Comment by jmaneyrol69** [comment, marked as answer]
Hi @Esiou, this discussion board is for suggestions and feature requests only.
Please submit a support request with Genymotion logs to our technical support at https://support.genymotion.com/hc/en-us/requests/new.
Thank you for your understanding.
